reactive/containerd.py

```diff
@@ -358,6 +370,37 @@ def publish_config():
     )
 
 
+@when('endpoint.untrusted.available')
+@when_not('untrusted.configured')
+@when_not('endpoint.containerd.departed')
+def untrusted_available():
+    """
+    Handle untrusted container runtime.
+
+    :return: None
+    """
+    untrusted_runtime = endpoint_from_flag('endpoint.untrusted.available')
+
+    DB.set('untrusted', dict(untrusted_runtime.get_config()))
+    config_changed()
+
+    set_state('untrusted.configured')
+
+
+@when('endpoint.untrusted.departed')
+def untrusted_departed():
+    """
+    Handle untrusted container runtime.
+
+    :return: None
+    """
+    DB.unset('untrusted')
+    DB.flush()
```

Comment:
we don't flush when we set, why do we need it here?

Reply:
I saw it on some other code and asked in IRC.  I'm trying to remember why, but there is a reason that unset can be async and needs a flush.